common/web/Makefile.mk

```diff
@@ -40,6 +40,11 @@ common_web_libolaweb_la_SOURCES = \
     common/web/SchemaParser.cpp \
     common/web/SchemaParser.h
 
+if USING_WIN32
+#Work around limitations with Windows library linking
+common_web_libolaweb_la_LIBADD = common/libolacommon.la
```

Comment:
I think this is correct. Windows is a bit funny about libraries from what I can tell and you have to declare the dependencies up front.  Linux allows you to put everything together at the end